src/components/FiltreGrup.jsx

```diff
@@ -160,15 +160,15 @@ const FiltreGrup = () => {
                   <div>
                     <input
                       type="checkbox"
-                      id="hibrid"
-                      name="hibrid"
-                      value="hibrid"
+                      id="Hybrid"
+                      name="Hybrid"
+                      value="Hybrid"
                       className="accent-background_green"
-                      checked={fields.remote.includes("hibrid")}
+                      checked={fields.remote.includes("Hybrid")}
                       onChange={(e) => handleCheckBoxChange(e, "remote")}
                     />
-                    <label htmlFor="hibrid" className="pl-1 cursor-pointer">
-                      Hibrid
+                    <label htmlFor="Hybrid" className="pl-1 cursor-pointer">
+                      Hybrid
```

Comment:
The label shoud be "Hibrid"

Reply:
I have updated the label from "Hybrid" to "Hibrid" as requested. Thank you for the feedback!